google-cloud-tools-plugin/google-cloud-core/src/com/google/cloud/tools/intellij/analytics/GoogleUsageTrackerService.java

```diff
@@ -103,7 +102,7 @@
    * Constructs a usage tracker configured with analytics and plugin name configured from its
    * environment.
    */
-  public GoogleUsageTracker() {
+  public GoogleUsageTrackerService() {
```

Comment:
So this constructor will be called for unit tests (since the service will be initialized with the instance by default), but now no actual operations are performed since we do all logic in `sendEvent` itself, correct?

Reply:
right. this is pretty much the same as how it worked before. If you look at the start of `GoogleUsageTrackerService#sendEvents` you'll see there is a guard for unitTestMode to pings from tests.